repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -82,12 +89,15 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
         }
 
         token = Some(uriTokens.remove(config.getTokenIndex))
-        wrappedRequest.setRequestURI(joinTokensIntoUri(uriTokens))
+        wrappedRequest.setRequestURI(joinTokensIntoPath(uriTokens))
       }
 
       filterChain.doFilter(wrappedRequest, wrappedResponse)
 
+      wrappedResponse.uncommit()
+
       val originalLocation = wrappedResponse.getHeader(CommonHttpHeader.LOCATION.toString)
+      // todo: What if the path is "/<tenant-id>"? The location header wouldn't be modified. Is that the correct behavior?
```

Comment:
doesn't seem correct to me, but it is current behavior. we should reach out to the original dev and see if he recalls why it was done this way